crates/telio-traversal/src/endpoint_providers/stun.rs

```diff
@@ -60,6 +60,15 @@ impl<Wg: WireGuard> StunEndpointProvider<Wg> {
             stun_peer_publisher,
         ))
     }
+
+    ///
```

Comment:
Write full comment :D

Reply:
Put it there to silence the editor warnings and forgot about it :see_no_evil: Fixed!